TradeItIosEmsApi/TradeItConnector.m

```diff
@@ -173,6 +181,7 @@ - (NSArray *)getLinkedLogins {
     for (NSDictionary *account in linkedAccounts) {
         [accountsToReturn addObject:[[TradeItLinkedLogin alloc] initWithLabel:account[@"label"]
                                                                        broker:account[@"broker"]
+                                                                       brokerLongName:account[@"brokerLongName"] ?: account[@"broker"]
```

Comment:
This defaults the value, but will that have any quirky UI side effects? Will this record be saved at a later time with the correct long name?

Reply:
The broker longname is passed when linking, so it will be updated the next time the user relink for existing accounts.